tests/Filter/FilterWithQueryBuilderTest.php

```diff
@@ -22,7 +22,7 @@ abstract class FilterWithQueryBuilderTest extends TestCase
     private $queryBuilder;
     private $expr;
 
-    public function setUp(): void
+    protected function setUp(): void
```

Comment:
This change should be catched before based on this rule:
https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/blob/eb321c2ecc333e4ebd63b64abd7a5cd1593515dc/.php_cs.dist#L39
:thinking: